lib/Doctrine/DBAL/Schema/ForeignKeyConstraint.php

```diff
@@ -357,7 +357,7 @@ public function onDelete()
      *
      * @param string $event Name of the database operation/event to return the referential action for.
      *
-     * @return string|null
+     * @return string|false|null
```

Comment:
`false|null` is undesirable.

Reply:
adding to ignore, but it's a bug, false is explicitly returned here :/